test.sh

```diff
@@ -17,6 +17,7 @@
 set -eu -o pipefail
 
 TESTSUITE="${TESTSUITE:-all}" # if env variable not set run all the tests
+CI="${CI:-false}" # if env variable not set don't run CI tests
```

Comment:
should set this ENV variable in github actions or Jenkins when we run it ?

Reply:
GitHub actions sets `CI=true` by default (https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables).

And is done in Jenkins by default as well https://github.com/jenkinsci/jenkins/pull/5370